packages/server/package.json

```diff
@@ -165,7 +165,7 @@
     "tslib": "^2.5.0",
     "tsx": "^3.12.7",
     "typescript": "^5.1.3",
-    "valibot": "^0.12.0",
+    "valibot": "^0.15.0",
```

Comment:
I thin we can remove Valibot from the `package.json` here. It is never imported.